dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/metadata/MetadataImportExportController.java

```diff
@@ -125,6 +137,12 @@
     @Autowired
     private ObjectFactory<GmlAsyncImporter> gmlAsyncImporterFactory;
 
+    @Autowired
+    protected ObjectMapper jsonMapper;
```

Comment:
I guess we can make both new fields `private`?

Reply:
fixed